packages/webapp/src/components/lists/TagsList/index.tsx

```diff
@@ -18,20 +17,17 @@ import { AddTagForm } from '~forms/AddTagForm'
 import { useWindowSize } from '~hooks/useWindowSize'
 import { EditTagForm } from '~forms/EditTagForm'
 import { Namespace, useTranslation } from '~hooks/useTranslation'
-import { TAG_CATEGORIES } from '~constants'
-import type { OptionType } from '~ui/ReactSelect'
+import type { CustomOption } from '~components/ui/CustomOptionsFilter'
 import { wrap } from '~utils/appinsights'
-import { createLogger } from '~utils/createLogger'
 import { useTagSearchHandler } from '~hooks/useTagSearchHandler'
 import { useMobileColumns, usePageColumns } from './columns'
-const logger = createLogger('tagsList')
 
 interface TagsListProps {
 	title?: string
 }
 
 export const TagsList: StandardFC<TagsListProps> = wrap(function TagsList({ title }) {
-	const { t, c } = useTranslation(Namespace.Tags)
+	const { t /*, c*/ } = useTranslation(Namespace.Tags)
```

Comment:
Should this comment just be removed now?

Reply:
I did in my other PR